modules/nw-egress-router-network-attachment.adoc

```diff
@@ -1,50 +0,0 @@
-// Module included in the following assemblies:
```

Comment:
Supporting egressrouter CR in v4.8, my understanding is we do not need manually setup  NetworkAttachmentDefinition object for egress router pod any more

Reply:
Agreed.  This file is deleted for 4.8.